### ISSUE TITLE
Train and Eval Set not available

### ISSUE THREAD

**Issue by LennartPiro** [wangzhecheng/DeepSolar#8]
Hi Wang ZheCheng,

I find I have access denied to the two links:
https://s3-us-west-1.amazonaws.com/roofsolar/SPI_train.tar.gz
https://s3-us-west-1.amazonaws.com/roofsolar/SPI_eval.tar.gz

Is there something I need to be doing to get access?
Access to https://s3-us-west-1.amazonaws.com/roofsolar/SPI_val.tar.gz is fine.

kind regards

_Originally posted by @james-skipper115 in https://github.com/wangzhecheng/DeepSolar/issues/4#issuecomment-453728217_

**Comment by beaucronin** [comment]
+1

**Comment by CoolSciGuy** [comment]
Hi folks,

Can you make train and Eval sets available? this would be much appreciated.

**Comment by vincentsohjh** [comment]
There is still an error. Please help.

**Comment by jameswelliott** [comment]
Am also experiencing this issue. Are there alternate links available?

**Comment by typicalTYLER** [comment]
> Seconded. Maybe you can consider uploading the files to [academictorrents.com](academictorrents.com) to reduce the S3 cost.

Hi all, I've added the 3 tar files comprising the dataset to academictorrents under this collection: http://academictorrents.com/collection/deepsolar-dataset

If @wangzhecheng wants me to take them down however, I can. It just seemed like an infrastructure failure because the validation tar was still up.

**Comment by rymanso** [comment]
Anyone manage to get around this hurdle? any help would be much appreciated

**Comment by LennartPiro** [comment]
Someone uploaded them to academictorrents.com and posted the link here - for some reason, however, those links disappeared again. But they are easy to find using the filenames:
http://academictorrents.com/browse.php?search=SPI_train
http://academictorrents.com/browse.php?search=SPI_eval

**Comment by rymanso** [comment]
didnt expect a response! thank you

**Comment by vincentsohjh** [comment]
The links not working. Anyone can help? Thank you!

**Comment by rymanso** [comment]
The link is working for me. Whats the problem that you have? Do you have a bit torrent client to download the files?

**Comment by ssakanbi** [comment]
Hello Guys are these links still working? I tried but got time out and not reachable. Please anyone help is much appreciated.

**Comment by ssakanbi** [comment]
Hello Ryanenian 
Kindly forward the working link here.
Thank you

**Comment by LennartPiro** [comment]
> 
> 
> Hello Guys are these links still working? I tried but got time out and not reachable. Please anyone help is much appreciated.

The website is down, I don't think anyone here can do anything about it. Just wait, I'm sure it will come back up. Or contact them on twitter https://twitter.com/academictorrent

**Comment by lsm12138** [comment]
> Someone uploaded them to academictorrents.com and posted the link here - for some reason, however, those links disappeared again. But they are easy to find using the filenames:
http://academictorrents.com/browse.php?search=SPI_train  
http://academictorrents.com/browse.php?search=SPI_eval

I tried these links, I can download them although it is very slow(nearly 50 kb/s) and not stable

**Comment by N1x0** [comment]
to anyone interested I will be seeding the `val`, `eval` and `train` datasets via the existing academictorrent for the next 2 weeks (or until I reach a 5x factor). So if you're interested in the data then now's the time :)

Edit:

I will also be validating both the TensorFlow and the PyTorch version for the current 3.7 version. Will take a bit but I'll upload everything once ready in case somebody is looking for that.

**Comment by ecmascriptguru** [comment]
@wangzhecheng , @jerluc 
Can you give us an example of the `eval_set_meta.csv`?

**Comment by satrya-sabeni** [comment]
> @wangzhecheng , @jerluc
> Can you give us an example of the `eval_set_meta.csv`?

I would also like to see an example of this

**Comment by ieee8023** [comment]
[Academic Torrents representative here]
I have added these torrents to two of our fast servers located in the US and Canada. Let me know if you need any help with torrents!

Here are direct links to the torrents:

SPI_train.tar.gz
https://academictorrents.com/details/8649f33cf8d661cbf3290bd4216c6ac637b777e0
SPI_eval.tar.gz
https://academictorrents.com/details/0a3344160f3bb1b6a82f2552cf8503d25e1b6b48
SPI_val.tar.gz
https://academictorrents.com/details/f0977a00ca9d61eefdfa232515ac6690d3b56fc5

**Comment by AloshkaD** [comment]
@ieee8023 Thanks, Joe! That's awesome.

**Comment by javierurien** [comment]
Hello, I am downloading the datasets, but for the train data I get too low throughput.

Would anyone seed for a while. Please? Many thanks!!
TIA!

**Comment by javierurien** [comment]
hey people! Thanks! It suddenly picked up to about 2mb/s!!
I'll seed as much as I can!!
Regards
